src/lib/mina_lib/mina_lib.ml

```diff
@@ -1496,7 +1496,7 @@ let start_filtered_log ~commit_id
     Ok () )
 
 let fetch_completed_snarks (module Context : CONTEXT) snark_pool network
-    top_block get_most_recent_valid_block =
+    top_block get_current_frontier =
```

Comment:
Suggest renaming `top_block` to `received_block` or `recently_received_block` otherwise it may be a bit unclear in the comparison below where we are checking if the node's frontier state has caught up to the most recent block received (minus 2).

Reply:
changed